platform/openide.text/src/org/netbeans/modules/openide/text/AskEditorQuestions.java

```diff
@@ -18,13 +18,31 @@
  */
 package org.netbeans.modules.openide.text;
 
+import java.util.MissingResourceException;
 import org.openide.DialogDisplayer;
 import org.openide.NotifyDescriptor;
 import org.openide.util.NbBundle;
+import org.openide.util.UserQuestionException;
 
 public final class AskEditorQuestions {
     private AskEditorQuestions() {
     }
+    
+    public static Boolean askUserQuestion(UserQuestionException uqe) {
```

Comment:
A nit: it might be nicer to use a 3-constant enum instead of a `Boolean`. The enum could say `YES`, `NO` and `ASK_USER`, making the result more obvious.